examples/config_example16.cfg

```diff
@@ -0,0 +1,56 @@
+my_value_string = "myvalue"
```

Comment:
config for testing various JSON constructs

Reply:
Not sure... It could be nice for some use-cases to consume/include a JSON file from a `.cfg` and reference its values or mix in with the flexi config tree. That said JSON allows constructs that are not valid in FlexiConfig (heterogeneous lists), so this probably requires some thought and discussion.